src/python/finalize/finalize.go

```diff
@@ -67,19 +73,22 @@ func (f *Finalizer) HandleCollectstatic() error {
 	if len(os.Getenv("DISABLE_COLLECTSTATIC")) > 0 {
 		return nil
 	}
-	if err := f.Command.Execute(f.Stager.BuildDir(), os.Stdout, os.Stderr, "pip-grep", "-s", "requirements.txt", "django", "Django"); err != nil {
-		return nil
-	}
 
-	managePyPath, err := f.ManagePyFinder.FindManagePy(f.Stager.BuildDir())
+	exists, err := f.Requirements.FindAnyPackage(f.Stager.BuildDir(), "django", "Django")
 	if err != nil {
 		return err
 	}
 
-	f.Log.Info("Running python %s collectstatic --noinput --traceback", managePyPath)
-	output := new(bytes.Buffer)
-	if err = f.Command.Execute(f.Stager.BuildDir(), output, text.NewIndentWriter(os.Stderr, []byte("       ")), "python", managePyPath, "collectstatic", "--noinput", "--traceback"); err != nil {
-		f.Log.Error(fmt.Sprintf(` !     Error while running '$ python %s collectstatic --noinput'.
+	if exists {
```

Comment:
I think we still need to return an error if `exists` is `false` to be equivalent with the old behavior (I think `pip-grep` used to return an error code in this case, but double-check me).

Reply:
Ah, nevermind. I see that the old implementation returns nil if `pip-grep` didn't find the package

Reply:
That said, I think it would be best to check if `exists` is `false`, then unindent the main code